barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.cpp

```diff
@@ -1767,28 +1692,33 @@ void UltraCircuitBuilder_<Arithmetization>::process_non_native_field_multiplicat
     // iterate over the cached items and create constraints
     for (const auto& input : cached_partial_non_native_field_multiplications) {
 
-        w_l().emplace_back(input.a[1]);
-        w_r().emplace_back(input.b[1]);
-        w_o().emplace_back(this->zero_idx);
-        w_4().emplace_back(input.lo_0);
+        blocks.main.w_l().emplace_back(input.a[1]);
+        blocks.main.w_r().emplace_back(input.b[1]);
+        blocks.main.w_o().emplace_back(this->zero_idx);
+        blocks.main.w_4().emplace_back(input.lo_0);
+        // blocks.main.populate_wires(input.a[1], input.b[1], this->zero_idx, input.lo_0);
```

Comment:
Are you planning on switching to what's in the comments?

Reply:
nope, deleting for now